app/src/interfaces/input-code/input-code.vue

```diff
@@ -278,6 +287,10 @@ function fillTemplate() {
 		emit('input', props.template);
 	}
 }
+
+function isInterpolation(value: any) {
+	return typeof value === 'string' && value.startsWith('{{') && value.endsWith('}}');
```

Comment:
We should probably make it strict (require value between brackets), for example we could re-use the pattern from https://github.com/directus/directus/blob/5a6071d88bc4b28e4b6899f17fe613e4d15a517a/packages/utils/shared/apply-options-data.ts#L38